server/utils/bookings/index.test.ts

```diff
@@ -220,22 +220,23 @@ describe('bookingUtils', () => {
       })
     })
 
-    it('generates a bespoke error when there is a conflicting lost bed', () => {
+    it('generates a bespoke error when there is a conflicting out-of-service bed', () => {
       const err = {
         data: {
-          detail: `Conflicting Lost Bed: ${lostBedId}`,
+          detail: `Conflicting out-of-service bed: ${lostBedId}`,
```

Comment:
Is this the actual error detail that comes back from the API?

Reply:
No, actually I abbreviated it. You're right - would probably be better with an actual cut'n'paste from the API.